releases/tests.py

```diff
@@ -157,6 +144,293 @@ def test_preview(self):
         self.assertEqual(Release.objects.preview().version, "1.9b2")
 
 
+class ReleaseTestCase(TestCase):
+    def test_is_published(self):
+        today = now().date()
+        future = today + datetime.timedelta(days=1)
+        past = today - datetime.timedelta(days=1)
+        cases = [
+            ({"date": None, "is_active": True}, False),
+            ({"date": None, "is_active": False}, False),
+            ({"date": today, "is_active": True}, True),
+            ({"date": today, "is_active": False}, False),
+            ({"date": past, "is_active": True}, True),
+            ({"date": past, "is_active": False}, False),
+            ({"date": future, "is_active": True}, False),
+            ({"date": future, "is_active": False}, False),
+        ]
+        for i, (params, expected) in enumerate(cases):
+            release = Release.objects.create(version=f"{i}.0", **params)
```

Comment:
The `is_published` property should work on unsaved models, so we don't even have to specify a version I think:

```suggestion
        for params, expected in cases:
            release = Release(**params)
```

Reply:
I will make another assertion for an unsaved model so we test correctness in both scenarios.